src/python/WMCore/WMSpec/Steps/Executors/DeleteFiles.py

```diff
@@ -157,7 +157,6 @@ def deleteOneFile(self, fileForTransfer, manager, waitTime):
             logging.error("Traceback: ")
             logging.error(traceback.format_exc())
             self.report.addError(self.stepName, 1, "StageOutFailure", str(msg))
```

Comment:
This exit code 1 is completely off here. How about we revive one exit code that was removed a couple of years ago `60313`, but I'd rephrase that to `Failed to delete file already staged in`, apparently that's what it's doing (I guess merge and logcollects).

Reply:
Failing to delete input files for LogCollect jobs after the other parts of the job are done is not an error we care about. As such, I think any instance of this error is caught upstream and ignored.

Not sure how Cleanup jobs are handled, if we actually fail them or not in cases there is a problem deleting files.

Reply:
You are right Dirk. It seems we always consider that as failure in the code. I guess that was a bug.
For clean up job, I think we should consider as failure but I am not sure whether we retry clean up jobs. Anyway, I will remove that and check the clean up jobs.